CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [CHANGE] Store-gateway: skip verifying index header integrity upon loading. To enable verification set `blocks_storage.bucket_store.index_header.verify_on_load: true`.
 * [CHANGE] Querier: change the default value of the experimental `-querier.streaming-chunks-per-ingester-buffer-size` flag to 256. #5203
 * [CHANGE] Querier: only initiate query requests to ingesters in the `ACTIVE` state in the ring. #5342
+* [FEATURE] Overrides-exporter: Add new metrics for write path and alertmanager.
```

Comment:
I'd say this is enhancement of existing feature, not a new feature.

We should add PR number `#5376` to the end of line, and we should also list new metric names. We need to mention that to activate these metrics, newly introduced option `-overrides-exporter.enabled-metrics` must explicitly list them.